.github/workflows/ci.yaml

```diff
@@ -49,7 +50,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.10"
+          python-version: "3.11"
```

Comment:
it will use python 3.11 to build the wheel,  but it will still be the same code that has been tested against all versions from 3.8.

And the wheel file will work in all pythons because of this: https://github.com/frequenz-floss/frequenz-sdk-python/blob/82fc84d1054188eb83a26205b927dc9462ddc270/pyproject.toml#L26